test/passing/invalid_docstring.ml.ref

```diff
@@ -0,0 +1,3 @@
+Warning: Invalid documentation comment:
+comment could not be parsed
```

Comment:
I think that because it's being fixed in odoc, this error message will depend on the version of odoc that's used. If that's the case it's better to just swallow the error.

Reply:
Do you suggest we return the comment as-is and treat it as a success?

Reply:
That's what I was suggesting, yes. We already do this when cinaps comment don't parse, for example.

But thinking a bit more about this, it's better to be consistent with the normal odoc warning case.

However, this means that the test will be flaky. Testing at the unit level instead would work, but I don't think we have a way to inject a doc comment parser at the moment. So I'm fine with removing the test as well.

Reply:
done